runtime/src/bank.rs

```diff
@@ -8623,8 +8623,8 @@ pub(crate) mod tests {
         let zero_lamports = 0;
         let little_lamports = 1234;
         let large_lamports = 123_456_789;
-        let some_slot = 1000;
-        let rent_collected = 22; // this is a function of 'some_slot'
+        let some_slot = MINIMUM_SLOTS_PER_EPOCH; // chosen to cause epoch to be +1
```

Comment:
iirc, `genesis_config` can return the real `slots_per_epoch` by calling some getter method? (yeah, blame me. i was lazy at the time...)

Reply:
chatted this over slack. i think it's fine as-is.

Reply:
// genesis_config.epoch_schedule.slots_per_epoch == 432_000 and is unsuitable for this test
unless we turn off warm up or something.